### PR TITLE
more integration fixes

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: talentmap_api/fsbid/services/client.py [hunk 1]
```diff
@@ -616,8 +619,8 @@ def fsbid_available_bidder_to_talentmap(data):
         "languages": fsbid_languages_to_tmap(data.get('languages', None)),
         "available_bidder_details": {
             **data.get("details", {}),
-            "is_shared": data.get('available_bidder_details', {}).get('is_shared') == '1',
-            "archived": data.get('available_bidder_details', {}).get('archived') == '1',
+            "is_shared": pydash.get(data, 'details.is_shared') == '1',
+            "archived": pydash.get(data, 'details.archived') == '1',
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/client.py:623]: 😭

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/client.py:623]: do you. not want to fallback to empty `{}`

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/client.py:623]: nvm, the fallback before was for details, not is_shared and is_archived

**Comment by elizabeth-jimenez** [comment]
@mjoyce91  Do we want to add the new url to `EXAMPLE_setup_environment.sh` and `deploy.sh`, as well. I know we still have to update the mock, and it may not be needed for this fix, but just in case it is, thought i'd mention it 🤷‍♀️

**Comment by mjoyce91** [comment]
@elizabeth-jimenez i never update either of those. But we should at some point

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - also, turns out mock actually does not need to be updated, since we define the endpoint in the environment variable

**Comment by scott062** [comment]
I think we just need the last nmn fix right?

**Comment by elizabeth-jimenez** [comment]
> I think we just need the last nmn fix right?

That's a thing? 😩

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍥 🍬 👍

**Pull request merged**